titus-supplementary-component/task-relocation-springboot/build.gradle

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2018 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+buildscript {
+    dependencies {
+        classpath "org.springframework.boot:spring-boot-gradle-plugin:2.1.1.RELEASE"
+    }
+}
+
+plugins {
+    id 'org.springframework.boot' version '2.1.1.RELEASE'
+}
+
+apply plugin: 'io.spring.dependency-management'
+
+configurations {
+    providedRuntime
```

Comment:
is this needed? I don't see it used in the `dependencies` closure.

Reply:
Removed.